test/tasks/main.cpp

```diff
@@ -63,7 +63,7 @@ void randomized_singlethreaded_execute(Tasks const& tasks, TaskGraph const& grap
 
         if (runTasksLeft != 0)
         {
-            TaskId const        randomTask  = rExec.tasksQueuedRun.at(rRand() % runTasksLeft);
+            TaskId const        randomTask  = rExec.tasksQueuedRun.begin()[std::ptrdiff_t(rRand()) % runTasksLeft];

```

Comment:
I'm curious of the benefits of `begin()[i]` vs just operator[]?

Reply:
the deprecation notice says to use begin()[], but it seems like it works so i'll switch to just operator[].